packages/sdks/src/blocks/video/video.lite.tsx

```diff
@@ -25,11 +25,54 @@ export default function Video(props: VideoProps) {
     },
   });
 
+  const videoRef = useRef<HTMLVideoElement>(null);
+  let lazyVideoObserver: IntersectionObserver | undefined;
+
+  onMount(() => {
+    if(props.lazyLoad){
+      lazyVideoObserver = new IntersectionObserver(function(entries) {
+        entries.forEach(function(entry) {
+          if (!entry.isIntersecting) return;
+
+          const videoElement = entry.target as HTMLVideoElement;
```

Comment:
you don't need `.target` when using mitosis element refs, they are directly available,

check: https://github.com/sidmohanty11/builder/blob/feat/variant-container-gen2/packages/sdks/src/blocks/embed/embed.lite.tsx/#L22

https://mitosis.builder.io/docs/hooks/#useref